src/features/game/expansion/Game.tsx

```diff
@@ -113,6 +122,78 @@ export const Game: React.FC = () => {
     );
   }
 
+  if (gameState.matches("landToVisitNotFound")) {
+    return (
```

Comment:
Consider moving this component elsewhere to keep Game.tsx focussed on the high level components.

Reply:
I have moved this down a little to be a conditional render of the `GameContent` function. There is really no other place for this to go. If a player trying to visit and no land is found then this rendered component IS the Game in that instance. This is basically the `visiting` state when no land is found. It has its on hud etc.